### ISSUE TITLE
Simple Carousel stops working unless I add the class 'fixed'

### ISSUE THREAD

**Issue by bartderijk** [blasten/turn.js#395]
So I implemented the Simple Carousel plugin. In this way I've got a gallery on a page which works but it only works as long as I give the page where its on the class 'fixed'. 

But adding the class 'fixed' makes the pages show up in places where it shouldn't so then I added a script which adds it only on the right pages. But then the carousel stopped working again :/ 

Any ideas on the cause of this? And maybe anyone can give me an hint on how to fix this? Thanks!

**Comment by punkpette** [comment]
Hi bartderik i found that turn.js kills all the events inside the flipbook.

i had an index page where i have links to pages in the flipbook, they work once but then stops working.

i saw that click event disappear. So this is a bug...

**Comment by TimOgilvy** [comment]
Yeah events definitely drop on both the free version and the paid version of turn.js

Do we know if someone has opened a ticket on that specific issue?

**Comment by punkpette** [comment]
found it!

using events like this:
//the event could be click... mouseenter, mouseleave, etc
//ITEM is the element that we want to add the event
$(document.body).on('EVENT', 'ITEM', function() {
//work some magic 
});

So you need to bind all the events like this :D

**Comment by TimOgilvy** [comment]
Do you reckon that counts as a bug still?  

Given that turn.js is angled as a jQuery plugin it should arguably carry all of the usual jQuery event bindings.

Great to have a workaround tho! I was re binding all my events in my 'turning' function to avoid this. It looks 'clean' in the code but I'm sure it slows down my turn on less powerful devices.

**Comment by punkpette** [comment]
Mmmm from what i understand now its not really a bug if you see some pages disappear from dom to improve performance so the content you had there is not really there
As in ajax the events no longer function because it was something that came up dinamically
With this in jquery we need to use on 

Liliana Iturribarría
Enviado desde mi iPhone
